### PR TITLE
Update ssam_se.py

### PR DESCRIPTION
**Pull request by stefan-jonasson**
Catch cases where the date is just a month.

### CODE REVIEW

File: custom_components/waste_collection_schedule/waste_collection_schedule/source/ssam_se.py [hunk 1]
```diff
@@ -79,12 +89,8 @@ def fetch(self):
...
                 if item["WasteType"] == "Trädgårdsavfall":
                     icon = "mdi:leaf"
-            found = 0
-            for x in entries:
-                if x.date == next_pickup_date and x.type == waste_type:
-                    found = 1
-            if found == 0:
-                entries.append(
-                    Collection(date=next_pickup_date, t=waste_type, icon=icon)
-                )
```
Comment (5ila5) [inline custom_components/waste_collection_schedule/waste_collection_schedule/source/ssam_se.py:89]: removing this duplicate check will sometimes return duplicate entries (second test):

it looks a bit janky but this approach seems to need some kind of duplicate filtering. You can prettify this code if you like but just removing it seems not to be a good solution.
```
Testing source ssam_se ...
  found 2 entries for Home
    2023-04-20 : Kärl 1, Kärl 370.0l [mdi:trash-can]
    2023-04-20 : Kärl 2, Kärl 370.0l [mdi:trash-can]
  found 12 entries for Bostadsrätt
    2023-04-11 : Hushållsavfall Kärl 600.0L [mdi:trash-can]
    2023-04-11 : Hushållsavfall Kärl 600.0L [mdi:trash-can]
    2023-04-11 : Hushållsavfall Kärl 600.0L [mdi:trash-can]
    2023-04-11 : Matpåsar [mdi:recycle]
    2023-04-11 : Matpåsar [mdi:recycle]
    2023-04-11 : Matpåsar [mdi:recycle]
    2023-04-10 : Trädgårdsavfall Kärl 370.0l [mdi:leaf]
    2023-04-10 : Trädgårdsavfall Kärl 370.0l [mdi:leaf]
    2023-04-10 : Trädgårdsavfall Kärl 370.0l [mdi:leaf]
    2023-04-10 : Matpåsar [mdi:recycle]
    2023-04-10 : Matpåsar [mdi:recycle]
    2023-04-10 : Matpåsar [mdi:recycle]
```

**Comment by 5ila5** [comment]
Does this happen regularly at some addresses? If so, would it be nice if you could provide one of those addresses as additional test case.

**Comment by stefan-jonasson** [comment]
It happens regularly,  test-case added.

**Comment by 5ila5** [comment]
Thanks for your contribution.

**Pull request merged**